public/index.html

```diff
@@ -24,7 +25,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Math Magicians</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
- [ ] In the public folder you can keep only `index.html` others are boilerplate
they are not needed now